goathacks/models.py

```diff
@@ -0,0 +1,52 @@
+from flask import flash, redirect, url_for
+from flask_login import UserMixin
+from sqlalchemy import Boolean, Column, DateTime, Integer, String
+from . import db
+from . import login
+
+class User(db.Model, UserMixin):
+    id = Column(Integer, primary_key=True)
+    email = Column(String, unique=True, nullable=False)
+    password = Column(String, nullable=False)
+    first_name = Column(String, nullable=False)
+    last_name = Column(String, nullable=False)
+    last_login = Column(DateTime, nullable=False)
+    active = Column(Boolean, nullable=False, default=True)
+    is_admin = Column(Boolean, nullable=False, default=False)
+    waitlisted = Column(Boolean, nullable=False, default=False)
+    shirt_size = Column(String, nullable=True)
+    accomodations = Column(String, nullable=True)
+    checked_in = Column(Boolean, nullable=False, default=False)
+    school = Column(String, nullable=True)
+    phone = Column(String, nullable=True)
+    gender = Column(String, nullable=True)
```

Comment:
@willhockey20 is this something that actually needs to be stored? I only included it bc it was in the original registration system.

Reply:
The data was required by mlh, still could be nice to have so I'd keep it